scripts/watchlist.js

```diff
@@ -1,36 +1,25 @@
 const watchlistResultsElement = document.getElementById("watchlist-results")
 const searchInput = document.getElementById("search-input")
 const searchBtn = document.getElementById("search-btn")
-let movieResults = JSON.parse(localStorage.getItem("Watchlist"))
-
-renderWatchlist()
 
 function renderWatchlist() {
+    const movieResults = JSON.parse(localStorage.getItem("Watchlist"))
```

Comment:
Similar to the index.js changes, we can change the `movieResults` variable to be scoped within the functions rather than keeping it in the global scope.

In this case, when `renderWatchlist` is called, we fetch the watchlist from localStorage, check if it's got any movies in it, and if so, pass the watchlist to the function that renders the HTML.